keys/keys.go

```diff
@@ -49,7 +52,8 @@ func New(c *deis.Client, id string, pubKey string) (api.Key, error) {
 	return key, nil
 }
 
-// Delete a key.
+// Delete a key. The key ID will be the key comment, usually the email or user@hostname
```

Comment:
the function name should be the noun, so here's a proposed alternative version: `Delete removes a key...`